src/main/webapp/config.js

```diff
@@ -0,0 +1,121 @@
+const toggleElementDisplay = (element, display) => {
```

Comment:
once i moved to the webapp folder i was allowed to use mostly normal js, previously forced only old js.

its all vanilla, tried to simplify as much as possible but might be some obvious improvements I've missed, will have a rereview aswell

Reply:
tried to only include comments when some unknown forces are in play, but if somethings unclear happy to add more.